src/main/scala/io/hydrosphere/serving/manager/domain/monitoring/MonitoringConfiguration.scala

```diff
@@ -0,0 +1,16 @@
+package io.hydrosphere.serving.manager.domain.monitoring
+
+import spray.json._
+
+case class MonitoringConfiguration(batch_size: Int = 100) {
+
+}
+
+
+object MCProtocol extends DefaultJsonProtocol {
```

Comment:
Weird object name.
Make object companion instead

Reply:
I was following spray tutorial https://github.com/spray/spray-json

Could you check a new version? If not, I can move it inside MonitoringConfiguration companion object.